src/ft2font_wrapper.cpp

```diff
@@ -1431,6 +1431,32 @@ PyFT2Font_get_image(PyFT2Font *self)
     return py::array_t<unsigned char>(dims, im.get_buffer());
 }
 
+const char *PyFT2Font__get_type1_encoding_vector__doc__ = R"""(
+    Return a list mapping CharString indices of a Type 1 font to FreeType glyph indices.
+
+    Returns
+    -------
+    list[int]
+)""";
+
+static std::array<FT_UInt, 256>
+PyFT2Font__get_type1_encoding_vector(PyFT2Font *self)
+{
+    auto face = self->x->get_face();
+    auto indices = std::array<FT_UInt, 256>{};
+    for (auto i = 0; i < indices.size(); ++i) {
+        auto len = FT_Get_PS_Font_Value(face, PS_DICT_ENCODING_ENTRY, i, nullptr, 0);
+        if (len == -1) {
+            throw std::runtime_error{
+                "FT_Get_PS_Font_Value tried to access a non-existent value"};
```

Comment:
Is it possible to give more context here, e.g. on which font and index the issue occurs, I could imagine one is quite lost otherwise.

Reply:
sure, done